### PR TITLE
Updated Domain operation permissions

### PR DESCRIPTION
**Pull request by Jackson-Woods**
Updated Domain operations pages to remove Directory.AccessAsUser.All permission and fill in other Domain.* scopes/app permissions.

### CODE REVIEW

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[e84da64](https://github.com/microsoftgraph/microsoft-graph-docs/commits/e84da640f80f05501e3850542feee1fe6503f99b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/domain-delete.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/beta/api/domain-delete.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-delete?view=graph-rest-beta&branch=pr-en-us-14149) (graph-rest-beta) |
[api-reference/beta/api/domain-forcedelete.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/beta/api/domain-forcedelete.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-forcedelete?view=graph-rest-beta&branch=pr-en-us-14149) (graph-rest-beta) |
[api-reference/beta/api/domain-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/beta/api/domain-get.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-get?view=graph-rest-beta&branch=pr-en-us-14149) (graph-rest-beta) |
[api-reference/beta/api/domain-list-domainnamereferences.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/beta/api/domain-list-domainnamereferences.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-list-domainnamereferences?view=graph-rest-beta&branch=pr-en-us-14149) (graph-rest-beta) |
[api-reference/beta/api/domain-list-serviceconfigurationrecords.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/beta/api/domain-list-serviceconfigurationrecords.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-list-serviceconfigurationrecords?view=graph-rest-beta&branch=pr-en-us-14149) (graph-rest-beta) |
[api-reference/beta/api/domain-list-verificationdnsrecords.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/beta/api/domain-list-verificationdnsrecords.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-list-verificationdnsrecords?view=graph-rest-beta&branch=pr-en-us-14149) (graph-rest-beta) |
[api-reference/beta/api/domain-list.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/beta/api/domain-list.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-list?view=graph-rest-beta&branch=pr-en-us-14149) (graph-rest-beta) |
[api-reference/beta/api/domain-post-domains.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/beta/api/domain-post-domains.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-post-domains?view=graph-rest-beta&branch=pr-en-us-14149) (graph-rest-beta) |
[api-reference/beta/api/domain-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/beta/api/domain-update.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-update?view=graph-rest-beta&branch=pr-en-us-14149) (graph-rest-beta) |
[api-reference/beta/api/domain-verify.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/beta/api/domain-verify.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-verify?view=graph-rest-beta&branch=pr-en-us-14149) (graph-rest-beta) |
[api-reference/v1.0/api/domain-delete.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/v1.0/api/domain-delete.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-delete?view=graph-rest-1.0&branch=pr-en-us-14149) (>= graph-rest-1.0) |
[api-reference/v1.0/api/domain-forcedelete.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/v1.0/api/domain-forcedelete.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-forcedelete?view=graph-rest-1.0&branch=pr-en-us-14149) (>= graph-rest-1.0) |
[api-reference/v1.0/api/domain-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/v1.0/api/domain-get.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-get?view=graph-rest-1.0&branch=pr-en-us-14149) (>= graph-rest-1.0) |
[api-reference/v1.0/api/domain-list-domainnamereferences.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/v1.0/api/domain-list-domainnamereferences.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-list-domainnamereferences?view=graph-rest-1.0&branch=pr-en-us-14149) (>= graph-rest-1.0) |
[api-reference/v1.0/api/domain-list-serviceconfigurationrecords.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/v1.0/api/domain-list-serviceconfigurationrecords.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-list-serviceconfigurationrecords?view=graph-rest-1.0&branch=pr-en-us-14149) (>= graph-rest-1.0) |
[api-reference/v1.0/api/domain-list-verificationdnsrecords.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/v1.0/api/domain-list-verificationdnsrecords.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-list-verificationdnsrecords?view=graph-rest-1.0&branch=pr-en-us-14149) (>= graph-rest-1.0) |
[api-reference/v1.0/api/domain-list.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/v1.0/api/domain-list.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-list?view=graph-rest-1.0&branch=pr-en-us-14149) (>= graph-rest-1.0) |
[api-reference/v1.0/api/domain-post-domains.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/v1.0/api/domain-post-domains.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-post-domains?view=graph-rest-1.0&branch=pr-en-us-14149) (>= graph-rest-1.0) |
[api-reference/v1.0/api/domain-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/v1.0/api/domain-update.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-update?view=graph-rest-1.0&branch=pr-en-us-14149) (>= graph-rest-1.0) |
[api-reference/v1.0/api/domain-verify.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/v1.0/api/domain-verify.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-verify?view=graph-rest-1.0&branch=pr-en-us-14149) (>= graph-rest-1.0) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C3%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202109031816376276-14149%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=55l%2FNeRepQraqYl6DEVR9epO1X%2F0zKiiTSHub7vIBZU%3D&st=2021-09-03T18%3A13%3A18Z&se=2021-10-04T18%3A18%3A18Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by mmacy** [comment]
+ @FaithOmbongi for review

File: api-reference/beta/api/domain-verify.md [hunk 1]
```diff
@@ -25,9 +25,9 @@ One of the following permissions is required to call this API. To learn more, in
 
 |Permission type      | Permissions (from least to most privileged)              |
 |:--------------------|:---------------------------------------------------------|
-|Delegated (work or school account) | Directory.Read.All    |
+|Delegated (work or school account) | Domain.Read.All, Domain.ReadWrite.All, Directory.Read.All    |
 |Delegated (personal Microsoft account) | Not supported.    |
-|Application | Directory.Read.All, Domain.ReadWrite.All |
+|Application | Domain.Read.All, Domain.ReadWrite.All, Directory.Read.All |
```
Comment (FaithOmbongi) [inline api-reference/beta/api/domain-verify.md:30]: @Jackson-Woods , kindly verify if both ***Read.All** permissions really work here. Also in the v1.0 topic. Thanks.

**Review by FaithOmbongi — COMMENTED** [review]
@Jackson-Woods , kindly verify if the verify action supports both ***Read.All** permissions as mentioned in the beta & v1.0 topics.

**Comment by kylemar** [comment]
You have cover all the one the GraphDocScanner found. Thanks!

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[4248601](https://github.com/microsoftgraph/microsoft-graph-docs/commits/4248601ab73cbcf8a7ac28c5209c5ce43f734c1e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/domain-delete.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/beta/api/domain-delete.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-delete?view=graph-rest-beta&branch=pr-en-us-14149) (graph-rest-beta) |
[api-reference/beta/api/domain-forcedelete.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/beta/api/domain-forcedelete.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-forcedelete?view=graph-rest-beta&branch=pr-en-us-14149) (graph-rest-beta) |
[api-reference/beta/api/domain-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/beta/api/domain-get.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-get?view=graph-rest-beta&branch=pr-en-us-14149) (graph-rest-beta) |
[api-reference/beta/api/domain-list-domainnamereferences.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/beta/api/domain-list-domainnamereferences.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-list-domainnamereferences?view=graph-rest-beta&branch=pr-en-us-14149) (graph-rest-beta) |
[api-reference/beta/api/domain-list-serviceconfigurationrecords.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/beta/api/domain-list-serviceconfigurationrecords.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-list-serviceconfigurationrecords?view=graph-rest-beta&branch=pr-en-us-14149) (graph-rest-beta) |
[api-reference/beta/api/domain-list-verificationdnsrecords.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/beta/api/domain-list-verificationdnsrecords.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-list-verificationdnsrecords?view=graph-rest-beta&branch=pr-en-us-14149) (graph-rest-beta) |
[api-reference/beta/api/domain-list.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/beta/api/domain-list.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-list?view=graph-rest-beta&branch=pr-en-us-14149) (graph-rest-beta) |
[api-reference/beta/api/domain-post-domains.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/beta/api/domain-post-domains.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-post-domains?view=graph-rest-beta&branch=pr-en-us-14149) (graph-rest-beta) |
[api-reference/beta/api/domain-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/beta/api/domain-update.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-update?view=graph-rest-beta&branch=pr-en-us-14149) (graph-rest-beta) |
[api-reference/beta/api/domain-verify.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/beta/api/domain-verify.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-verify?view=graph-rest-beta&branch=pr-en-us-14149) (graph-rest-beta) |
[api-reference/v1.0/api/domain-delete.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/v1.0/api/domain-delete.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-delete?view=graph-rest-1.0&branch=pr-en-us-14149) (>= graph-rest-1.0) |
[api-reference/v1.0/api/domain-forcedelete.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/v1.0/api/domain-forcedelete.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-forcedelete?view=graph-rest-1.0&branch=pr-en-us-14149) (>= graph-rest-1.0) |
[api-reference/v1.0/api/domain-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/v1.0/api/domain-get.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-get?view=graph-rest-1.0&branch=pr-en-us-14149) (>= graph-rest-1.0) |
[api-reference/v1.0/api/domain-list-domainnamereferences.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/v1.0/api/domain-list-domainnamereferences.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-list-domainnamereferences?view=graph-rest-1.0&branch=pr-en-us-14149) (>= graph-rest-1.0) |
[api-reference/v1.0/api/domain-list-serviceconfigurationrecords.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/v1.0/api/domain-list-serviceconfigurationrecords.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-list-serviceconfigurationrecords?view=graph-rest-1.0&branch=pr-en-us-14149) (>= graph-rest-1.0) |
[api-reference/v1.0/api/domain-list-verificationdnsrecords.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/v1.0/api/domain-list-verificationdnsrecords.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-list-verificationdnsrecords?view=graph-rest-1.0&branch=pr-en-us-14149) (>= graph-rest-1.0) |
[api-reference/v1.0/api/domain-list.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/v1.0/api/domain-list.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-list?view=graph-rest-1.0&branch=pr-en-us-14149) (>= graph-rest-1.0) |
[api-reference/v1.0/api/domain-post-domains.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/v1.0/api/domain-post-domains.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-post-domains?view=graph-rest-1.0&branch=pr-en-us-14149) (>= graph-rest-1.0) |
[api-reference/v1.0/api/domain-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/v1.0/api/domain-update.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-update?view=graph-rest-1.0&branch=pr-en-us-14149) (>= graph-rest-1.0) |
[api-reference/v1.0/api/domain-verify.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-fix-domain-permissions/api-reference/v1.0/api/domain-verify.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/domain-verify?view=graph-rest-1.0&branch=pr-en-us-14149) (>= graph-rest-1.0) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C3%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202109032147377921-14149%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=f2rUNJapPiWFynueg%2FI1lhm%2B0b1NlTNXYn9wbrtZJmI%3D&st=2021-09-03T21%3A44%3A15Z&se=2021-10-04T21%3A49%3A15Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Review by FaithOmbongi — APPROVED** [review]
Thanks @Jackson-Woods for raising this fix.

**Pull request merged**